samples/boards/stm32/pm_lptim/src/main.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2020 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+#include <string.h>
+#include <soc.h>
+#include <device.h>
+#include <drivers/gpio.h>
+#include <sys/printk.h>
+
+const struct device *gpio_bn, *gpio_led;
+
+bool ready_to_sleep;
+
+/* in msec */
+/* setting to K_TICKS_FOREVER willl activate the deepsleep mode */
+#define SLEEP_TIME 11000
```

Comment:
11 sec is a bit long (I sometime worry the use case is broken). Maybe 3 sec would be enough ?

Reply:
yes, reduced (but still modifiable by the user)